pyam/utils.py

```diff
@@ -329,21 +329,28 @@ def convert_r_columns(c):
     df.loc[df.unit.isnull(), "unit"] = ""
 
     # verify that there are no nan's left (in columns)
-    null_rows = df.isnull().values
+    null_rows = df.isnull().T.any()
```

Comment:
This is not duplicated, because `df.isnull()` returns a matrix of bools - the first `T.any()` reduces this to "is there any null in a row", the next line asks "is there any null in any row" - and prints out the offending row(s) if so.

Reply:
Ah ok I see, then looks good to me.